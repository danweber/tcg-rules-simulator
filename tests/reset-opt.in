VERSION:2
# fusion evo resets once-per-turn on ESS effect
FIELD1:Blue,CS2-Colossal,CS2-01 Yellow,CS1-Obsidian,CS1-01
FIELD2:Zero,CS2-02,REST One,CS2-02,REST Two,CS2-05,REST Three,CS2-04,REST 
HAND1:CS2-Rampager
DECK1:DW1-01 DW1-01
GAME:P1 T4 MAIN 7
DUMP:HAND,DP

TITLE:Once-per-turn resets after fusion evo
TEXT:Attack with inherited effect in 1 monster, fusion evo into another, should be able to use once-per-turn again

CMD:ATTACK Yellow One
DUMP:HAND,DP

ECHO: fusion evo, do when evolving

# hey are we ignoring that newlabel??
CMD:EVO CS2-Rampage Yellow 0 Blue NewLabel X1
STEP:AUTOANSWER:CAPTURE
DUMP:DP


ECHO: ------ now attack into other, get +1000 again
CMD:ATTACK I7 Three
EFFECT-TREE

DUMP:DP
EXIT
