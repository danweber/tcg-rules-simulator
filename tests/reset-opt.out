FIELD1:Blue,CS2-12,CS2-01,7500DP Yellow,CS1-10,CS1-01,13000DP
FIELD2:Zero,CS2-02,REST,2000DP One,CS2-02,REST,2000DP Two,CS2-05,REST,5500DP Three,CS2-04,REST,4500DP
HAND1:CS2-Rampager
HAND2:
GAME:P1 T4 MAIN 7
FIELD1:Blue,CS2-12,CS2-01,7500DP Yellow,CS1-10,CS1-01,REST,14000DP
FIELD2:Zero,CS2-02,REST,2000DP Two,CS2-05,REST,5500DP Three,CS2-04,REST,4500DP
HAND1:CS2-Rampager
HAND2:
GAME:P1 T4 MAIN 7
ECHO: fusion evo, do when evolving
FIELD1:I7,CS2-Rampager,CS1-10,CS1-01,CS2-12,CS2-01,14000DP
FIELD2:Three,CS2-04,REST,4500DP
GAME:P1 T4 MAIN 7
ECHO: ------ now attack into other, get +1000 again
EFFECT-TREE:NOTHING
FIELD1:I7,CS2-Rampager,CS1-10,CS1-01,CS2-12,CS2-01,REST,15000DP
FIELD2:
GAME:P1 T4 MAIN 7
